sherpa-ncnn/jni/jni.cc

```diff
@@ -290,10 +239,13 @@ JNIEXPORT jlong JNICALL Java_com_k2fsa_sherpa_ncnn_SherpaNcnn_new(
   sherpa_ncnn::ModelConfig model_config =
       sherpa_ncnn::GetModelConfig(env, _model_config);
 
+  sherpa_ncnn::DecoderConfig decoder_config;
```

Comment:
Could you follow the way to initialize `model_config` to initialize
`decoder_config`?